### PR TITLE
Improve fix for parallel build

### PR DESCRIPTION
**Pull request by olamy**
- **Simply copying dependencies using maven dependency plugin**

follow up of #294 with a better fix

### CODE REVIEW

File: applications/proberapp/pom.xml [hunk 1]
```diff
@@ -211,39 +211,23 @@
 
         <plugins>
             <plugin>
-                <groupId>com.coderplus.maven.plugins</groupId>
-                <artifactId>copy-rename-maven-plugin</artifactId>
-                <version>1.0.1</version>
+                <groupId>org.apache.maven.plugins</groupId>
```
[the remark this answers is not in the document]
Comment (ludoch) [reply applications/proberapp/pom.xml:214]: But then, if scope provided, would it still be built before the final probber web app, when using parallel builds?

[the remark this answers is not in the document]
Comment (ludoch) [reply applications/proberapp/pom.xml:214]: Finally, I think we should also update this md doc: https://github.com/GoogleCloudPlatform/appengine-java-standard/blob/main/TRYLATESTBITSINPROD.md (this is the process to copy the runtime jars somewhere at the top of the built web app war directory so they can be used by the entrypoint defined in the appengine-web.xml file

[the remark this answers is not in the document]
Comment (olamy) [reply applications/proberapp/pom.xml:214]: The idea here is not changing anything to what was done before but using a different (probably more reliable) technique to copy those jars.
The result is really exactly the same except this is not copying anymore those jars from the output directory of another non related maven module.

**Review by ludoch — APPROVED** [review]

**Comment by lachlan-roberts** [comment]
I can confirm that this fixes #293 for me.

**Review by lachlan-roberts — APPROVED** [review]

File: applications/proberapp/pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (olamy) [reply applications/proberapp/pom.xml:214]: oh right. that's bad as those jars are now in `WEB-INF/lib` 
Sorry for that, I will propose another PR.

[the remark this answers is not in the document]
Comment (olamy) [reply applications/proberapp/pom.xml:214]: Sorry my bad, I didn't read your comments correctly. 
Definitely better to not have those jars in `WEB-INF/lib` 
better using `provided` see PR https://github.com/GoogleCloudPlatform/appengine-java-standard/pull/298

**Pull request merged**